apollo-router/src/router_factory.rs

```diff
@@ -142,6 +142,28 @@ async fn create_plugins(
         }
     }
 
+    // The "include_subgraph_errors" plugin must always be active. If it is not
+    // configured, add an instance with default configuration to ensure that
+    // subgraph errors are correctly redacted.
+    let subgraph_error_plugin = "experimental.include_subgraph_errors".to_string();
+    if !plugin_instances
+        .iter()
+        .any(|(x, _)| x == &subgraph_error_plugin)
+    {
+        match plugin_registry.get(&subgraph_error_plugin) {
+            Some(factory) => match factory.create_instance(&serde_json::json!({})).await {
+                Ok(plugin) => {
+                    plugin_instances.push((subgraph_error_plugin, plugin));
+                }
+                Err(err) => errors.push(ConfigurationError::PluginConfiguration {
+                    plugin: subgraph_error_plugin,
+                    error: err.to_string(),
+                }),
+            },
+            None => errors.push(ConfigurationError::PluginUnknown(subgraph_error_plugin)),
```

Comment:
if it is explicitely not in the plugin registry, should we even show an error?

Reply:
I'm just preserving the existing behaviour here. It seems reasonable since we are letting people know their config is invalid since they've asked for something which doesn't exist and they should correct their config.